Makefile

```diff
@@ -79,13 +79,16 @@ BUILD_TAG ?=
 HV_CFG_LOG = $(HV_OUT)/cfg.log
 VM_CONFIGS_DIR = $(T)/misc/config_tools
 ASL_COMPILER ?= $(shell which iasl)
+DPKG_BIN ?= $(shell which dpkg)
```

Comment:
Can we generalize checking for uninstalled or version dependencies early in the processing, maybe in another PR?  Issue #7095 tracks improved reporting of missing dependencies.

Reply:
I agree we should have a more generic mechanism in place. This one is a slightly special case though because:
1. `dpkg` is not an explicit dependency unlike the other tools that are all mentionned in the GSG. Note that we shouldn't add it to the GSG either because not having it does not prevent you from building ACRN, it only prevents you from creating the Debian package which is not a problem per se (especially since if `dpkg` is not there, it's precisely because you're not on a Debian system)
2. The alternative to this patch in my opinion, would be to remove the fact that the Debian package is created automatically by default and we make it an explicit target that the user would call, e.g. `make deb-pkg` (as an alternative to `make targz-pkg`. This explicit target was my first choice, but I'm fine with this being done automatically without asking the user too... provided we don't error out if that part (only) fails.

How about we take this patch in, and refactor the whole thing when we have time to address #7095 in a more systemic and generic fashion (something I do not have time to do now)

Reply:
Works for me @gvancuts